From 67a2f42b011d95e856210cbe662252234bd9d5a4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 2 Feb 2017 13:07:27 -0800
Subject: [PATCH] TOOLS-1673 sdcadm getAgentImages not working due to missing
 name attribute Reviewed by: Julien Gilli <julien.gilli@joyent.com>

---
 lib/steps/agent-services.js | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/steps/agent-services.js b/lib/steps/agent-services.js
index 38e8c70..afd2df0 100644
--- a/lib/steps/agent-services.js
+++ b/lib/steps/agent-services.js
@@ -73,6 +73,7 @@ function agentServicesEnsureCreated(arg, cb) {
     agentNames.forEach(function (n) {
         var logLevelKey = n.toUpperCase().replace('-', '_') + '_LOG_LEVEL';
         agentServices[n] = {
+            name: n,
             type: 'agent',
             params: {
                 tags: {
-- 
2.21.0

